enhancers_crons.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/getsentry/sentry-go"
+	batchv1 "k8s.io/api/batch/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func runCronsEnhancer(ctx context.Context, scope *sentry.Scope, pod *v1.Pod, sentryEvent *sentry.Event) error {
+
+	// get owningCronJob if exists
+	owningCronJob, err := getCronJob(ctx, pod, scope)
+	if err != nil {
+		return err
+	}
+	if owningCronJob == nil {
+		return errors.New("cronjob: pod not created under a cronjob")
+	}
+
+	scope.SetContext("monitor", sentry.Context{"slug": owningCronJob.Name})
+
+	sentryEvent.Fingerprint = append(sentryEvent.Fingerprint, owningCronJob.Kind, owningCronJob.Name)
+
+	setTagIfNotEmpty(scope, "cronjob_name", owningCronJob.Name)
+
+	// add breadcrumb with cronJob timestamps
+	scope.AddBreadcrumb(&sentry.Breadcrumb{
+		Message:   fmt.Sprintf("created cronjob %s", owningCronJob.Name),
+		Level:     sentry.LevelInfo,
+		Timestamp: owningCronJob.CreationTimestamp.Time,
+	}, breadcrumbLimit)
+
+	metadataJson, err := prettyJson(owningCronJob.ObjectMeta)
+
+	if err == nil {
+		scope.SetExtra("Cronjob Metadata", metadataJson)
+	} else {
+		return err
+	}
+	return nil
+}
+
+func getCronJob(ctx context.Context, pod *v1.Pod, scope *sentry.Scope) (*batchv1.CronJob, error) {
```

Comment:
To make it more explicit, let's rename it to `getCronJobParent` or similar

Reply:
I renamed it to `getOwningCronJob` to stay consistent